tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
I guess we also need to tell TS to generate a `index.d.ts` file for the repo as well (in the root directory)